data/frm/Crecy_1997_xPonthus/stemma.gv

```diff
@@ -0,0 +1,23 @@
+digraph {
+
+x [color="grey"];
+x -> P1; 
+x -> P2; 
+x -> P3; 
+x -> P4; 
+
+x-> 1;
+1 [label="", color="grey"];
+1 -> G; 
+1 -> P5;
+
+x -> 2;
```

Comment:
`x²` ?

Reply:
X2 indicates a groupe of mss. (c), composed by CF, PR, A, L2, whose ancestor is not represented in the stemma. I encoded this letter as a generic node 2.